src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php

```diff
@@ -36,10 +67,10 @@ class TemplateController implements ContainerAwareInterface
      */
     public function templateAction($template, $maxAge = null, $sharedAge = null, $private = null)
     {
-        if ($this->container->has('templating')) {
-            $response = $this->container->get('templating')->renderResponse($template);
-        } elseif ($this->container->has('twig')) {
-            $response = new Response($this->container->get('twig')->render($template));
+        if ($this->templating) {
+            $response = $this->templating->renderResponse($template);
+        } elseif ($this->twig) {
```

Comment:
Let's check Twig first :)

Reply:
That'd be bc breaking changes...

Reply:
I agree with this one, but not the other one.